Plugin Radioactive Cloud/src/main/java/net/simon987/pluginradioactivecloud/RadioactiveWorldUtils.java

```diff
@@ -0,0 +1,77 @@
+package net.simon987.pluginradioactivecloud;
+
+import net.simon987.server.game.world.TileMap;
+import net.simon987.server.game.world.TilePlain;
+import net.simon987.server.game.world.World;
+import org.bson.types.ObjectId;
+
+import java.awt.*;
+import java.util.ArrayList;
+import java.util.Random;
+
+public class RadioactiveWorldUtils {
+
+  /**
+   * Generate a list of biomass blobs for a world
+   */
+  public static ArrayList<RadioactiveObstacle> generateRadioactiveObstacles(World world, int minCount, int maxCount) {
+
+    Random random = new Random();
+    int radioactiveObjCount = random.nextInt(maxCount - minCount) + minCount;
+    ArrayList<RadioactiveObstacle> radioactiveObstacles = new ArrayList<>(radioactiveObjCount);
+
+    // Count number of plain tiles. If there is less plain tiles than desired amount
+    // of radioactive objects, set the desired amount of radioactive objects to the
+    // plain tile count
+    TileMap m = world.getTileMap();
+    int plainCount = 0;
```

Comment:
This exact code block is used twice, we should extract it into its own function (e.g. WorldUtils.countTiles(int tileId))